src/bring.ts

```diff
@@ -217,6 +217,25 @@ class Bring {
         } // endCatch
     } // endSaveItem
 
+    /**
+     *   Save an image to an item
+     *
+     *   @param formData The formdata you want to send.
+     *   @param itemUuid The itemUUID you want to update.
+     *   returns an imageUrl and answerHttpStatus should contain 204. If not -> error
+     */
+    async saveItemImage(itemUuid: string, formData: { [key: string]: any } | undefined): Promise<{ imageUrl: string }> {
```

Comment:
Can you provide an example of the formData, then we can maybe improve the typings if that this will be self-explanatory.  And what does the API perform if I pass undefined?

Reply:
Right... I just took over the typings of formData from request-promise. So unsure if undefined would return anything useful.
Theoretically the way I tested it, one could send any data to bring and it will return a url afaik. I just tested with random filetypes.

```typescript
response = await bringUploadItemImage(selectedItem, {
	clientInstanceId: "clientInstanceId",
	imageData: (await superagent.get("https://example.com/url-to-image.jpg").buffer(true).parse(superagent.parse.image)).body
})
```

I also tested it with the readstream... that works too:

```typescript
response = await bringUploadItemImage(selectedItem, {
	clientInstanceId: "clientInstanceId",
	imageData: fs.createReadStream("C:\\image.jpg") // something like this.
})
```

Reply:
Thanks a lot for the explanation, this helps a lot. So `imageData` lets us provide our own image and is necessary for the call. And what is the purpose of `clientInstanceId`?

Reply:
So imageData lets us provide our own image and is necessary for the call.
- Exactly
And what is the purpose of clientInstanceId?
- I honestly have no clue. I just saw it get sent exactly like that all the time in the requests. Like not an id... but the string "clientInstanceId". I don't even know if it is required... I'll see if I can test without.

Reply:
I have already checked, it is not necessary, but if done by official API client, we can send it. However, I prefer such things to be handled internally as this is nothing, that the user of our API wrapper should take care of. I have adjusted types and fixed the remove method, as I missed that the headers were not passed during review. You can check the latest commits.